.babelrc

```diff
@@ -1,15 +1,35 @@
 {
```

Comment:
Some lovely day we will gather the power to upgrade to babel 7 so this config can be less verbose, today is not that day...